toolchain/sem_ir/typed_insts.h

```diff
@@ -868,6 +868,25 @@ struct ImplDecl {
   DeclInstBlockId decl_block_id;
 };
 
+// A symbolic witness that a type implements an interface. It represented a
```

Comment:
Matching "implements", feels like the same tense should be used here.

```suggestion
// A symbolic witness that a type implements an interface. It represents a
```